src/Lists/List.tsx

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components';
+import ListItem from './ListItem';
+import { fromTheme } from '../Theme';
+
+export interface ListProps {
+  /**
+   * Should only contain ListItems
+   */
+  children: ListItem | ListItem[];
+  /**
+  * Whether this list is ordered (true) or unordered (false)
+  */
+  ordered?: boolean;
+}
+
+/**
+* Renders either an ordered or unordered list component, depending on the value
+* of the "ordered" prop. Both list types are styled identically, but the
+* difference is important semantically
+*/
+
+const List = styled.ol.attrs(
+  ({ ordered }: ListProps) => (
+    {
+      as: ordered ? 'ol' : 'ul',
+    }
+  )
+)<ListProps>`
+  list-style: none;
+  > li {
+    border-top: ${fromTheme('border', 'hairline')};
+    &:first-child {
+      border-top: none;
+    }
```

Comment:
I'm wondering how much flexibility we should give to the user in terms of styling by allowing them to specify whether they want the default list styles to show and whether they want the separating border via boolean props for each of these two cases. I suppose we could come back to expand the props that could be provided and offer more flexibility in the future too, but maybe it would be easier to do this upfront.

Reply:
In principle, yes, we want to be flexible, but I also think there's some value to enforcing consistency through the design system (back in my magazine days I used to lose my mind over how many different ways people styled lists in articles). So I'd rather avoid adding options to cover hypothetical cases and create something that works for what we need, since we're the primary users of mark-one.

Reply:
Ok good point - we can always come back and adjust if needed.